### PR TITLE
Enhance GitHub actions

### PR DESCRIPTION
**Pull request by blootsvoets**
- Use buildx for multiplatform build
- Add standard docked container labels
- add release workflow
- test curl and app runtime docker availability
- Avoid docker pull limit by logging in with secrets `DOCKERHUB_USERNAME` and `DOCKERHUB_TOKEN`

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -93,8 +97,86 @@ jobs:
...
+
+      - name: Check backend docker image
+        run: >
+          docker run --rm ${{ env.DOCKER_IMAGE_BACKEND }}:${{ steps.docker_meta_backend.outputs.version }} curl --help &&
+          docker run --rm ${{ env.DOCKER_IMAGE_BACKEND }}:${{ steps.docker_meta_backend.outputs.version }} authorizer-app-backend --help
+
+      # If the image was pushed, we need to pull it again to inspect it
+      - name: Push backend image
+        if: ${{ github.event_name != 'pull_request' }}
+        run: docker push ${{ env.DOCKER_IMAGE }}:${{ steps.docker_meta_backend.outputs.version }}
```
Comment (mpgxvii) [inline .github/workflows/main.yml:177]: Should this be a `pull` instead of `push`?

Comment (blootsvoets) [reply .github/workflows/main.yml:177]: Push is intentional, the build part only loads the image into the local docker environment. Indeed, the env variable is a copy paste error.

Comment (blootsvoets) [reply .github/workflows/main.yml:177]: I see I also missed the comment above, I’ll update it.

**Review by mpgxvii — APPROVED** [review]
LGTM.

**Pull request merged**